CMakeLists.txt

```
@@ -174,7 +182,7 @@ install( TARGETS date
 export( TARGETS date NAMESPACE date:: FILE dateTargets.cmake )
 if (CMAKE_VERSION VERSION_LESS 3.15)
     install(
-        FILES include/date/date.h
+        FILES "${TARGET_HEADERS}"
```

Quoted string makes CMake treat the list as a single file:

```suggestion
FILES ${TARGET_HEADERS}
```